app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job so far. Please kindly ensure  `:confirmable ` is also enabled in the user model. This enables the application to asks for confirmation of email. it is important as it verifies if an account is already confirmed to sign in, and to send emails with confirmation instructions. please Don't forget to activate it in the devise migration file as well.
- [ ] please remove the generated comments in the user model for cleaner code practices. 😄